docs/configuration/index.rst

```diff
@@ -34,6 +34,10 @@ Defines parameters for the API:
   Defaults to True.
 - init_mgmt_timeout: Timeout to wait for device to apply changed management IP.
   Defaults to 30, specified in seconds (integer).
+- mgmtdomain_reserved_count: Number of IP addresses to reserve for internal use on
+  each defined management domain when assigning new management IP addresses to devices.
+  Defaults to 5 (e.g. meaning 10.0.0.0 through 10.0.0.4 would remain unassigned on
+  a domain for 10.0.0.0/24).
```

Comment:
Something might have changed with the new code here because before the first 5 _host_ addresses were reserved, eg 1-5 so the first one that was assigned to a switch was .6

Reply:
Apologies, @indy-independence  - nothing has changed about the code. I wrote out the docs from memory, but my memory was faulty. The new code does still count 10.0.0.0 as *0*, so 10.0.0.0 through 10.0.0.5 is reserved. I'll update the docs I wrote.